fix(ast-tools): use oxfmt to format generated code

camc314 opened the pull request:
https://github.com/oxc-project/oxc/commit/a8e5181ab73e3cee5904af3acc6c5927def9d500#diff-dc1a48ba0d3878494d547c7c5fb975720dddac4966c510f9babec50161110538 broke ast tools generation.

This PR uses oxfmt to format the generated ast files

leaysgur reviewed (approved):
Ah, sorry! I wasn’t aware of this usecase.

Thanks & it seems good to me.
(Using oxc_formatter crate instead of creating tmp file may be possible but it seems too much?)

camc314 committed:
fix(ast-tools): use oxfmt to format generated code (#15064)

https://github.com/oxc-project/oxc/commit/a8e5181ab73e3cee5904af3acc6c5927def9d500#diff-dc1a48ba0d3878494d547c7c5fb975720dddac4966c510f9babec50161110538 broke ast tools generation.

This PR uses oxfmt to format the generated ast files

camc314 commented:
> (Using oxc_formatter crate instead of creating tmp file may be possible but it seems too much?)

I thought about it, but it means we have to compile more code here, which would proably slow down CI even more.
